purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/PaywallAnimation.kt

```diff
@@ -43,8 +34,14 @@ class PaywallAnimation(
 }
 
 @OptIn(InternalRevenueCatAPI::class)
-object AnimationTypeAsObjectSerializer : EnumAsObjectSerializer<PaywallAnimation.AnimationType>(
-    enumClass = PaywallAnimation.AnimationType::class,
```

Comment:
Hmm just to check how would `EnumAsObjectSerializer` handle receiving a string value instead of an object? Just wondering if we would be falling back to the default paywall in that case...

Reply:
It actually fell back to the default that was passed in.